src/Npgsql/Npgsql.csproj

```diff
@@ -18,8 +18,9 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' OR '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="System.Memory" Version="4.5.3" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.3" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
```

Comment:
BTW why not 1.1.1?

Reply:
It requires `System.Threading.Tasks.Extensions` version 4.5.4, but this one requires package updates. Not sure that it's a good idea for a fix.

Reply:
Should probably make sure that 1.1.0 fixes the actual issue (i.e. contains DisposeAsync)...

Reply:
From [NuGet for version 1.1.0](https://www.nuget.org/packages/Microsoft.Bcl.AsyncInterfaces/1.1.0):

> Provides the IAsyncEnumerable<T> and IAsyncDisposable interfaces and helper types for .NET Standard 2.0. This package is not required starting with .NET Standard 2.1 and .NET Core 3.0.
>
> Commonly Used Types:
> System.IAsyncDisposable
> System.Collections.Generic.IAsyncEnumerable
> System.Collections.Generic.IAsyncEnumerator

Reply:
OK, looks good.